addons/mail/models/mail_message.py

```diff
@@ -574,7 +573,8 @@ def create(self, values_list):
         tracking_values_list = []
         for values in values_list:
             if 'email_from' not in values:  # needed to compute reply_to
-                values['email_from'] = self._get_default_from()
+                author_info = self.env['mail.thread']._message_compute_author(values.get('author_id'), None, raise_exception=False)
```

Comment:
I guess you pass the `None` for clarity, even if not needed?

Reply:
Indeed